ext/src/DefaultSession.c

```diff
@@ -847,17 +857,66 @@ PHP_METHOD(DefaultSession, prepare)
     timeout = PHP5TO7_ZVAL_MAYBE_P(opts->timeout);
   }
 
-  future = cass_session_prepare_n((CassSession *)self->session->data,
-                                  Z_STRVAL_P(cql), Z_STRLEN_P(cql));
+  if (self->persist) {
```

Comment:
and that's not mine 🙃 it's the prepared statement support
but these changes are backward compatibles since everything is enclosed by a few `self->persist` checks, which defaults to 0

Reply:
sadly, they didn't add any integration tests about it 😞

Reply:
nevermind, it doesn't defaults to 0, and these persistent prepared statement will be enabled :

<img width="961" alt="image" src="https://user-images.githubusercontent.com/717869/158150605-8fe840ca-d5ca-46ea-9bd4-9ae80a5ecc5b.png">

Reply:
sure
![image](https://user-images.githubusercontent.com/1446309/160001407-75289e7a-c9cf-41c4-9d97-02be925e3d6a.png)